### PR TITLE
The Next Web Clone

### PR DESCRIPTION
**Pull request by kunmi02**
The is a website clone of the next web. We worked on 2 breakpoints which are 768px and 1024px.

Please view on 1024px  screen and below.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1684 @@
...
+                <span> &copy;2006-2020 The Next Web B.V.</span>
+              </p>
+
+              <p class="amsterdam">Made with &lt;3 in Amsterdam.</p>
+              <span>Powered by</span>
+              <a href="#" class="stackpath">STACKPATH</a>
+            </div>
+          </div>
+        </div>
+      </footer>
```
Comment (uryelah) [inline index.html]: - [x] [OPTIONAL] Ideally the `<footer>` tag will be outside the `<main>` tag. The `<main>` tag should contain the main content, the focus, of the page, the footer is usually not the main content of a webpage. Read more about semantic tags and their meaning [here](https://www.w3schools.com/html/html5_semantic_elements.asp).

File: css768.css [hunk 2]
```diff
@@ -0,0 +1,207 @@
+body {
+  margin: 0;
+  padding: 0;
+  color: #555;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
```
Comment (uryelah) [inline css768.css]: - [x] Let's avoid repetition and keep the styles that stay constant for the body in just the general CSS file, no need to repeat it for each screen size.

File: css1024.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+.image1 {
+  grid-area: 1 / 1 / 3 / 3;
+  background-position: center;
+  padding: 0;
+  height: 100%;
+  width: 100%;
+}
+
+.image2 {
+  height: 300px;
```
Comment (uryelah) [inline css1024.css]: - [ ] For the bigger than 1024px screens this height should change to accommodate the growth of image1.

File: css1024.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  color: #555;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  max-width: 1024px;
```
Comment (uryelah) [inline css1024.css]: - [x] Is this really necessary? Why don't you just make the body that 100% of the page regardless of it's size in the main CSS file? 
Ex:
`max-width: 100vw`
Try to write styles that require the least amount of changes to be responsive, that will make your CSS files lighter.

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi **Akshay Narkar**, I'm Sarah your reviewer for this PR 👩‍🚀 

You did a great job building the small and medium views of this project, the attention to the hover effects and typography was especially good, but it seems that the project layout is breaking if viewed on bigger screens than ~1030px.

Read the review comments below carefully and take some time to compare your clone with the [original page](https://thenextweb.com/), using your browser responsiveness tools, and then get to coding! You are really close, just really need to fix the large windows layout issue.

If you think you need a refresher about how to use responsiveness tools I'd check [this Firefox, the best browser for developers, turorial](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode).

> Note: It seems that this project was submitted only by Akshay and marked as 'I worked alone on this project' even though it seems like two people built it. If this was by mistake make sure to contact student success to make sure both of you get the correct status for this project completion.

Rock on! 🎸 👨‍🎤 

# Required Changes 🏗️ 🐛 💡

-----

 **Kindly fix the pointed out items and comments below before requesting the next code review**

- [x] Write your next [commit messages following the best practices](https://chris.beams.io/posts/git-commit/), by capitalizing them and writing them in the present/imperative.

- [x] The page body is not taking the full width of the window. Make sure your clone also shows up properly on large screens, >1024px, with 100% width. Check how in the screenshot two points below there's a white margin at the right side of it.

- [x] Make sure also that the page layout is also the same on the large screens:
  - Having the TNW logo and a red-ish gradient over the main article's image from the 3 articles at the top.
  - Having the 'Latest Funding' section showing 4 articles in the same row.
  - Having the 'Categories (Apps, Gear, Tech, etc)' section aligning 3 boxes per row.
  - The two smaller articles at the top of the page should be aligned top and bottom with the bigger one, the two smaller ones should have the same height.

**Compare the origin site on larger screens and your clone in the screenshots below:**

 <table style="width:100%">
  <tr>
    <th>Original Page</th>
    <th>Clone</th>
  </tr>
  <tr>
    <td valign="top"></td>
    <td></td>
  </tr>
</table> 

- [NOTE] If you are having issues with your media specific CSS files right now try to use media queries to make the site responsive instead. [Check this W3Schools article on media queries.](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)

- [x] Instead of repeating styles between the 3 stylesheets, for example: `font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;`, just set those styles common to all of the sizes in one main CSS. Try to avoid repetition, in general, to keep a DRY CSS code and have lighter stylesheet files that load faster.


- [x] [OPTIONAL] Let's add some box-shadow under the fixed top navbar to make it look closer to the original one and make it 'pop' in the eye by giving it some depth.


------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @kunmi02 and @akshay-narkar :wave: 

### Thank you for making some of the requested changes, but we still need to improve on some aspects. Please check my suggestions below based on the last review:

- [x] Please update your Live Link every time you make a change to your project, OR even better: when you paste your `index.html` file link there, take the second generated link to the right, it auto-updates as you make the changes (takes some minutes tho)!

- [x] In your first three articles, please increase the font-size of what's written there, decrease the height of your images and align them, and add the TNW logo, just like in the original.

- [x] In your 'Latest Funding' section on Large Screens, you are currently displaying only 3 articles, whilst in the original, there are 4 articles.

- [x] In your Categories section on Large Screens, you are currently displaying only 1 article per row, whilst in the original, there are 3 articles per row.

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by uryelah — APPROVED** [review]
Hi **Azeez Azeez & Akshay Narkar**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job adding that large screen responsiveness!

You now have the minimum required to continue to the next project, but before that make sure to check the optional suggestions left below, they'll make your site look more professional and hopefully improve your CSS abilities at that.
You can work on them in your spare time or just look up the suggestions for learning's sake.

Good luck with your next HTML&CSS projects! 🚀  

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---------

### Optional suggestions

- [x] [OPTIONAL] Let's fix that overlapping text over the image on medium to smaller screens. That makes your page look amatourish and decreases how accessible and easy to read the text is if someone has vision problems. 
>Tip: Add a [`min-height` property](https://css-tricks.com/almanac/properties/m/min-height/) to the divs with class `.hardfork-latest` in such way that their minimum height is the height of the image they contain in the right.

Screenshot_2020-10-30 TNW

- [x] [OPTIONAL] The gradient logo should take 100% height of the image it's overlapping. 
Change the height property of the `.tnw-overlay` div so it takes the necessary height. 
>TIp: [Use your browser inspector tool to test CSS changes there before actually changing the file directly.](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Debugging_CSS)

 Screenshot_2020-10-30 TNW(1)

---------

    
    
**Keep being an awesome dev and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**